Sources/SimpleCurrencyConverter/CurrencyConverterProtocol.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+
+public protocol CurrencyConverterProtocol {
+    
+    /// This function is used to retrieve the exchange rate for a given base currency and a list of target currencies.
+    ///
+    /// This function uses the network manager to make a request to an exchange rates API, passing the base currency code and the target currency codes as parameters.
+    /// If the request is successful, the returned exchange rates are passed to the resolve closure.
+    /// If the request fails, the error is passed to the reject closure.
+    ///
+    ///  - Parameter baseCurrency: A Currency object representing the base currency for which the exchange rate is desired.
+    ///  - Parameter targetCurrencies: An array of Currency objects representing the target currencies for which the exchange rates are desired.
+    ///  - Parameter resolve: A closure that takes in a dictionary of exchange rates, where the keys are the currency codes of the target currencies and the values are the exchange rates (expressed as a Double)
+    ///  - Parameter reject: A closure that takes in an error object, it will be called if an error occurs while trying to get the exchange rates.
+            
+    func getExchangeRate(baseCurrency: Currency, targetCurrencies: [Currency],
+                         resolve: @escaping ([String: Double]) -> Void,
+                         reject: @escaping (Error) -> Void)
```

Comment:
@ts-dzmitry-kurski Instead of using `resolve` and `reject`, can you return `Result` in only one closure?
Same comment for functions signature below.